### PR TITLE
Coding convention applied split 2-7-2 (of 2-14-2)

### PR DESCRIPTION
**Pull request by Quix0r**
This PR is the next of **14** upcoming splits of the first series (yes, a lot has been changed) which started with #3254 (origin: #3112 ). It includes coding-convention stuff like added spaces and curly braces. Also some `dbm::is_result()` may have been included.

The commit has been split out from a larger commit containing 192 (!) changed files.

More similar changes: (in order they are created)

- [x] part 2 #3254 
- [x] part 2-1-2 #3287
- [x] part 2-2-2 #3369
- [x] part 2-3-2 #3375
- [x] part 2-4-2 #3381
- [x] part 2-5-2 #3469 
- [x] part 2-6-2 #3473 
- [ ] part 2-7-2 (this)
- [ ] part 2-8-2 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split2-8-2
- [ ] part 2-9-2 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split2-9-2
- [ ] part 2-10-2 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split2-10-2
- [ ] part 2-11-2 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split2-11-2
- [ ] part 2-12-2 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split2-12-2
- [ ] part 2-13-2 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split2-13-2
- [ ] part 2-14-2 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split2-14-2
- [ ] part 2-2 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split2-2 (**final!**)
- [ ] part 3 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split3
- [ ] part 4 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split4
- [ ] part 5 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split5
- [ ] part 6 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split6
- [ ] / : https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-root
- [ ] utils/ : https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-utils
- [ ] mod/ https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-mod
- [ ] include 1: https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-include-if
- [ ] include 2: https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-include-foreach
- [ ] others: https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-others
- [ ] else-if: https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-else-if

Unrelated but comes next:
- [ ] chmod https://github.com/Quix0r/friendica/tree/fixes/chmod

### CODE REVIEW

File: include/expire.php [hunk 1]
```diff
@@ -5,44 +5,18 @@
...
 
-	if (($argc == 2) && ($argv[1] == 'delete')) {
-		logger('Delete expired items', LOGGER_DEBUG);
-		// physically remove anything that has been deleted for more than two months
-		$r = dba::p("SELECT `id` FROM `item` WHERE `deleted` AND `changed` < UTC_TIMESTAMP() - INTERVAL 60 DAY");
-		while ($row = dba::fetch($r)) {
-			dba::delete('item', array('id' => $row['id']));
-		}
-		dba::close($r);
+	$r = q("DELETE FROM `item` WHERE `deleted` = 1 AND `changed` < UTC_TIMESTAMP() - INTERVAL 60 DAY");
```
Comment (tobiasd) [inline include/expire.php]: I think this should use the new DB functions, and not the old one. But @annando should have a look as well.

File: include/items.php [hunk 2]
```diff
@@ -2240,7 +2241,7 @@ function drop_item($id, $interactive = true) {
 		}
 
 		// If item has attachments, drop them
-		foreach (explode(", ", $item['attach']) as $attach) {
```
Comment (tobiasd) [inline include/items.php]: Just thinking, would not be one space too much in the $matches now, that the explode is not including the space after the comme?

File: include/queue.php [hunk 3]
```diff
@@ -29,6 +29,25 @@ function queue_run(&$argv, &$argc) {
 		// Handling the pubsubhubbub requests
 		proc_run(array('priority' => PRIORITY_HIGH, 'dont_fork' => true), 'include/pubsubpublish.php');
 
+		$interval = ((get_config('system','delivery_interval') === false) ? 2 : intval(get_config('system','delivery_interval')));
+
+		// If we are using the worker we don't need a delivery interval
```
Comment (tobiasd) [inline include/queue.php]: Why is this included? The worker is the only background processor we have nowadays.

File: include/salmon.php [hunk 4]
```diff
@@ -149,14 +150,6 @@ function slapper($owner, $url, $slap) {
 	if ($return_code > 299) {
 		logger('compliant salmon failed. Falling back to old status.net');
 
-		// Last try. This will most likely fail as well.
```
Comment (tobiasd) [inline include/salmon.php]: Why is this removed?

File: include/tags.php [hunk 5]
```diff
@@ -116,10 +116,10 @@ function create_tags_from_itemuri($itemuri, $uid) {
 
 function update_items() {
 
-	$messages = dba::p("SELECT `oid`,`item`.`guid`, `item`.`created`, `item`.`received` FROM `term` INNER JOIN `item` ON `item`.`id`=`term`.`oid` WHERE `term`.`otype` = 1 AND `term`.`guid` = ''");
```
Comment (tobiasd) [inline include/tags.php]: @annando which DB functions should be used here?

File: include/text.php [hunk 6]
```diff
@@ -2220,6 +2221,10 @@ function text_highlight($s, $lang) {
 		$lang = 'javascript';
 	}
 
+	if (! strpos('Text_Highlighter', get_include_path())) {
```
Comment (tobiasd) [inline include/text.php]: Why was this block added here?

File: src/Network/Probe.php [hunk 7]
```diff
@@ -1063,7 +1063,8 @@ private function diaspora($webfinger) {
 	private function ostatus($webfinger) {
 		$data = array();
 		if (is_array($webfinger["aliases"])) {
-			foreach ($webfinger["aliases"] as $alias) {
```
Comment (tobiasd) [inline src/Network/Probe.php]: This "as" is replaced with a "AS" and a comment saying it should be a lowercase "as"

File: include/Contact.php [hunk 8]
```diff
@@ -788,22 +790,25 @@ function account_type($contact) {
...
 		$type = ACCOUNT_TYPE_PERSON;
+	}
 
 	// The "contact-type" (contact table) and "account-type" (user table) are more general then the chaos from above.
-	if (isset($contact["contact-type"]))
+	if (isset($contact["contact-type"])) {
 		$type = $contact["contact-type"];
-	if (isset($contact["account-type"]))
+	}
+	if (isset($contact["account-type"])) }
```
Comment (annando) [inline include/Contact.php]: That ```}``` surely should be a ```{```

File: include/datetime.php [hunk 9]
```diff
@@ -139,7 +139,7 @@ function datetime_convert($from = 'UTC', $to = 'UTC', $s = 'now', $fmt = "Y-m-d
 	 * months and days always start with 1.
 	 */
 
-	if (substr($s,0,10) <= '0001-01-01') {
+	if (substr($s, 0, 10) == '0000-00-00') {
```
Comment (annando) [inline include/datetime.php]: is this a merge failure? Please restore the previous check with ```<= '0001-01-01'```

File: include/datetime.php [hunk 10]
```diff
@@ -178,11 +178,11 @@ function datetime_convert($from = 'UTC', $to = 'UTC', $s = 'now', $fmt = "Y-m-d
 function dob($dob) {
 	list($year,$month,$day) = sscanf($dob,'%4d-%2d-%2d');
 
-	$f = get_config('system', 'birthday_input_format');
+	$f = get_config('system','birthday_input_format');
 	if (! $f) {
 		$f = 'ymd';
 	}
-	if ($dob <= '0001-01-01') {
+	if ($dob === '0000-00-00') {
```
Comment (annando) [inline include/datetime.php]: This seems to be some old code as well. Please restore this as well.

File: include/dbstructure.php [hunk 11]
```diff
@@ -1762,7 +1762,7 @@ function dbstructure_run(&$argv, &$argc) {
 	global $a, $db;
 
 	if (is_null($a)) {
-		$a = new App(dirname(__DIR__));
+		$a = new App;
```
Comment (annando) [inline include/dbstructure.php]: This looks like some old code as well. Please restore ```$a = new App(dirname(__DIR__));```

File: include/diaspora.php [hunk 12]
```diff
@@ -946,11 +946,11 @@ private static function post_allow($importer, $contact, $is_comment = false) {
 			// Yes, then it is fine.
 			return true;
 		// Is it a post to a community?
-		} elseif (($contact["rel"] == CONTACT_IS_FOLLOWER) && ($importer["page-flags"] == PAGE_COMMUNITY)) {
+		} elseif (($contact["rel"] == CONTACT_IS_FOLLOWER) && ($importer["page-flags"] == PAGE_COMMUNITY) || $is_comment) {
```
Comment (annando) [inline include/diaspora.php]: I don't like combinations of different logical operators without braces. They tend to confuse. Why not use the old code here?

File: include/diaspora.php [hunk 13]
```diff
@@ -946,11 +946,11 @@ private static function post_allow($importer, $contact, $is_comment = false) {
...
 			// That's good
 			return true;
 		// Is the message a global user or a comment?
```
Comment (annando) [inline include/diaspora.php]: When you change the code here, you should change the comment line as well.

File: include/expire.php (hunk 1 above)
Comment (annando) [inline include/expire.php]: You obviously reverted parts of the last expire changes. Could you restore them again?

File: include/items.php [hunk 14]
```diff
@@ -2240,7 +2241,7 @@ function drop_item($id, $interactive = true) {
...
+		foreach (explode(",", $item['attach']) as $attach) {
```
Comment (annando) [inline include/items.php]: With the space after the ```,``` I would like it more. I guess this was done intentionally.

File: include/pgettext.php [hunk 15]
```diff
@@ -44,7 +44,7 @@ function get_browser_language() {
 	// check if we have translations for the preferred languages and pick the 1st that has
 	for ($i = 0; $i < count($lang_list); $i++) {
 		$lang = $lang_list[$i];
-		if ($lang === 'en' || (file_exists("view/lang/$lang") && is_dir("view/lang/$lang"))) {
+		if (file_exists("view/lang/$lang") && is_dir("view/lang/$lang")) {
```
Comment (annando) [inline include/pgettext.php]: Why have you reverted this code?

File: include/poller.php [hunk 16]
```diff
@@ -20,14 +20,16 @@
 function poller_run($argv, $argc){
 	global $a, $db, $poller_up_start, $poller_db_duration;
 
-	$poller_up_start = microtime(true);
-
-	$a = new App(dirname(__DIR__));
+	if (is_null($a)) {
```
Comment (annando) [inline include/poller.php]: it seems as if here you had reverted some programming as well. I guess this wasn't done intentionally?

File: include/queue.php (hunk 3 above)
Comment (annando) [inline include/queue.php]: I guess - like several parts above - this is some merge problem that reverted some stuff.

File: include/queue.php [hunk 17]
```diff
@@ -29,6 +29,25 @@ function queue_run(&$argv, &$argc) {
...
+		if ($r) {
+			foreach ($r as $rr) {
+				logger('queue: deliverq');
+				proc_run(PRIORITY_HIGH,'include/delivery.php',$rr['cmd'],$rr['item'],$rr['contact']);
+				if ($interval) {
+					time_sleep_until(microtime(true) + (float) $interval);
+				}
+			}
+		}
+
```
Comment (annando) [inline include/queue.php]: This also looks like some old code.

File: include/salmon.php [hunk 18]
```diff
@@ -149,14 +150,6 @@ function slapper($owner, $url, $slap) {
...
-		$xmldata = array("me:env" => array("me:data" => $data,
-				"@attributes" => array("type" => $data_type),
-				"me:encoding" => $encoding,
-				"me:alg" => $algorithm,
-				"me:sig" => $signature3,
-				"@attributes2" => array("key_id" => $keyhash)));
-
```
Comment (annando) [inline include/salmon.php]: I guess you removed this by accident?

File: include/socgraph.php [hunk 19]
```diff
@@ -1239,6 +1239,10 @@ function poco_check_server($server_url, $network = "", $force = false) {
 			} else {
 				$register_policy = REGISTER_CLOSED;
 			}
+
+			if (isset($data->version)) {
+				$last_contact = datetime_convert();
+			}
```
Comment (annando) [inline include/socgraph.php]: Why was this added?

File: include/socgraph.php [hunk 20]
```diff
@@ -2028,6 +2032,7 @@ function get_gcontact_id($contact) {
 		return false;
 	}
 
+	/// @TODO backward-compatibility or old-lost code?
```
Comment (annando) [inline include/socgraph.php]: This is intentional. So you can remove the todo.

File: include/socgraph.php [hunk 21]
```diff
@@ -2156,6 +2161,7 @@ function update_gcontact($contact) {
 
 	$fields["hide"] = $r[0]["hide"];
 
+	/// @TODO backward-compatibility or old-lost code?
```
Comment (annando) [inline include/socgraph.php]: Same goes here.

File: include/socgraph.php [hunk 22]
```diff
@@ -2164,7 +2170,7 @@ function update_gcontact($contact) {
 	fix_alternate_contact_address($contact);
 
 	if (!isset($contact["updated"])) {
-		$contact["updated"] = dbm::date();
+		$contact["updated"] = datetime_convert();
```
Comment (annando) [inline include/socgraph.php]: ```dbm::date``` had been here intentionally.

File: include/tags.php (hunk 5 above)
Comment (annando) [inline include/tags.php]: I guess this was some more merge problem.

File: include/tags.php [hunk 23]
```diff
@@ -116,10 +116,10 @@ function create_tags_from_itemuri($itemuri, $uid) {
...
+	$messages = $db->q("SELECT `oid`,`item`.`guid`, `item`.`created`, `item`.`received` FROM `term` INNER JOIN `item` ON `item`.`id`=`term`.`oid` WHERE `term`.`otype` = 1 AND `term`.`guid` = ''", true);
 
-	logger("fetched messages: ".dba::num_rows($messages));
-	while ($message = dba::fetch($messages)) {
+	logger("fetched messages: " . count($messages));
```
Comment (annando) [inline include/tags.php]: This file looks like you accidentally restored some old version.

File: include/text.php [hunk 24]
```diff
@@ -654,7 +654,7 @@ function get_template_file($a, $filename, $root = '') {
 	$theme = current_theme();
 
 	// Make sure $root ends with a slash /
-	if ($root !== '' && substr($root, -1, 1) !== '/') {
+	if ($root !== '' && $root[strlen($root)-1] !== '/') {
```
Comment (annando) [inline include/text.php]: I like the ```substr``` more.

File: src/Network/Probe.php (hunk 7 above)
Comment (annando) [inline src/Network/Probe.php]: Maybe some merge problem again?

File: include/expire.php (hunk 1 above)
Comment (Quix0r) [inline include/expire.php]: I don't know why this got reverted ... :-(

Comment (annando) [inline include/expire.php]: There was reverted really much stuff.

Comment (Quix0r) [inline include/expire.php]: Strange that it happens. I rebased my commits on upstream/master as always.

Comment (Quix0r) [inline include/expire.php]: In my `develop` branch, nothing like that happened.

**Comment by Quix0r** [comment]
I think this was the best choice. I will come back with a redone rewrite. :)

File: include/expire.php (hunk 1 above)
Comment (annando) [inline include/expire.php]: Like I always say: Rebase is bad :)

Comment (Quix0r) [inline include/expire.php]: No, it is not. Merges will cause the history to become more complex than before. Rebase is a chance to review your code. A rebase rewrites history until where it has been done one but that can be mitigated with `git push --force`.
